apps/earn-protocol/components/molecules/SidebarElements/OrderInfoWithdraw.tsx

```diff
@@ -1,26 +1,96 @@
+  }
+
+  const { fromAmount, slippage, priceImpact, toAmount } = transaction.metadata
+
   return (
     <div className={orderInfoDepositWithdrawStyles.depositViewWrapper}>
-      <Icon tokenName={vault.inputToken.symbol as TokenSymbolsList} size={64} />
+      <Icon tokenName={fromAmount.token.symbol.toUpperCase() as TokenSymbolsList} size={64} />
       <Text variant="h2">{formatCryptoBalance(amountParsed)}</Text>
       <Text variant="p2semi">{amountDisplayUSD}</Text>
+      <Box className={orderInfoDepositWithdrawStyles.depositDetails}>
+        <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsTitle}>
+          Changes & Fees
+        </Text>
+        <SimpleGrid columns={2} gap={2}>
+          <Text variant="p3semi">Withdraw Amount</Text>
+          <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
+            {formatCryptoBalance(fromAmount.amount)} {fromAmount.token.symbol}
+          </Text>
+          {toAmount && priceImpact && (
+            <>
+              <Text variant="p3semi">Swap</Text>
+              <div style={{ display: 'flex', alignItems: 'center', justifyContent: 'flex-end' }}>
+                <Icon
+                  tokenName={fromAmount.token.symbol.toUpperCase() as TokenSymbolsList}
+                  size={20}
+                />
+                <Text
+                  variant="p3semi"
+                  className={orderInfoDepositWithdrawStyles.depositDetailsValue}
+                >
+                  {formatCryptoBalance(fromAmount.amount)} {'-> '}
+                </Text>
+                <Icon
+                  tokenName={toAmount.token.symbol.toUpperCase() as TokenSymbolsList}
+                  size={18}
+                />
+                <Text
+                  variant="p3semi"
+                  className={orderInfoDepositWithdrawStyles.depositDetailsValue}
+                >
+                  {formatCryptoBalance(toAmount.amount)}
+                </Text>
+              </div>
+              <Text variant="p3semi">Price Impact</Text>
+              <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
+                {formatCryptoBalance(priceImpact.price.value)}{' '}
+                {(priceImpact.price.quote as IToken).symbol} (
+                {formatPercent(priceImpact.impact.value, {
+                  precision: 2,
+                })}
+                )
+              </Text>
+              <Text variant="p3semi">Slippage</Text>
+              <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
+                {formatPercent(slippage.value, { precision: 2 })}
+              </Text>
+            </>
+          )}
+          <Text variant="p3semi">Transaction Fee</Text>
+          <Text variant="p3semi" className={orderInfoDepositWithdrawStyles.depositDetailsValue}>
+            {transactionFeeLoading
+              ? 'Loading...'
```

Comment:
same comment here regarding skeleton and n/a

Reply:
thanks

Reply:
done